node-firefox-scripts/launch-firefoxos.js

```diff
@@ -13,7 +13,7 @@ var launchApp = require('node-firefox-launch-app');
 var pushApp = require('push-app');
 
 // Where our app is located
-var appPath = path.join(__dirname, '../testrunner');
+var appPath = path.join(__dirname, process.argv[2]);
```

Comment:
This is the cause of Grunt weirdness.

`var appPath = path.join(__dirname, process.argc >= 3 ? process.argv[2] : '../testrunner');`